app/controllers/application_controller.rb

```diff
@@ -17,4 +17,19 @@ def default_url_options
     return {} if I18n.locale == I18n.default_locale
     { locale: I18n.locale }
   end
+
+  def has_news_feature?
```

Comment:
@anthotsang Let's put the `has_news_feature?` and `get_ld_user` method definitions within a `protected` section instead of `private` to expose them to child controllers and keep things DRY.